connect/lib/core/src/main/scala/io/iohk/atala/connect/core/service/ConnectionServiceImpl.scala

```diff
@@ -129,6 +133,8 @@ private class ConnectionServiceImpl(
           // TODO: According to the standard, we should rather use 'pthid' and not 'thid'
           thid = invitation.id,
           label = None,
+          goalCode = None,
+          goal = None,
```

Comment:
As per the ticket description, `goalCode` and `goal` here should be set to the values found in the invitation, and not `None`.